docs/rfcs/fix-overlapping-bars-on-continuous-axes.md

```diff
@@ -0,0 +1,37 @@
+# RFC: Fix overlapping bars on continuous axes
+
+## How x-axis ticks are rendered?
+
+Let TR be the total width available to render the bars. The first bar should start from the beginning of TR and the last bar should end at the end of TR. The corresponding x-axis ticks should be rendered at the center of the bars. The first tick should align with the center of first and the last tick should align with the center of the last bar. So the x-axis scale range becomes [BW/2, TR-BW/2] where BW is the bar width. d3 linear scale dont account for the bar width. So if we know the bar width we can just shift the x-axis scale range by BW/2 [0, TR-BW] so that the scale return start point of each bar. So we first need to know the bar width in order to render the bars and the x-axis ticks correctly. 
```

Comment:
add the diagram also that you have in the document

Reply:
done